packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -316,13 +314,6 @@ BaseLightningElementConstructor.prototype = {
                     )}: 1 argument required, but only 0 present.`
```

Comment:
I am also tempted with removing the argument length check as well, this is also a duplicate.